q2_qemistree/assets/index.html

```diff
@@ -29,22 +29,26 @@ <h1>
 </h1>
 
 <div class='row'>
-  <div class='col-md-4'>
+  <div class='col-md-3'>
     <a href='./qemistree.tree' target="_blank" rel="noopener noreferrer" class='btn btn-default'>
-      Tree
+      Qemistree
     </a>
   </div>
-  <div class='col-md-4'>
+  <div class='col-md-3'>
     <a href='./labels.tsv' target="_blank" rel="noopener noreferrer" class='btn btn-default'>
       Labels
     </a>
   </div>
-  <div class='col-md-4'>
+  <div class='col-md-3'>
     <a href='./colors.tsv' target="_blank" rel="noopener noreferrer" class='btn btn-default'>
       Colors
     </a>
   </div>
+  <div class='col-md-3'>
```

Comment:
Since the barplot is optional, this button should only be displayed if the user inputs a table. You will need to use jinja directives like `{% if .... %}` to omit certain HTML sections. See these docs:

https://jinja.palletsprojects.com/en/2.11.x/templates/#list-of-control-structures

Reply:
How would one use jinja directives here?
`{% if "./barplots.tsv" %}` ??

Reply:
You would need to pass another variable to the environment. Namely in line 206 of _plot.py. Instead of having:

```python
q2templates.render(index_path, output_dir, context={'url': url})
```

It should probably be:

```python
q2templates.render(index_path, output_dir, context={'url': url, 'has_barplots': feature_table is not None})
```

And then in the template you can check `{% if has_barplots .....`. Does that make sense?

Reply:
yes, thank you! 🎉

Reply:
👍